lib/typescript/render/providers/google/GoogleRender.tsx

```diff
@@ -51,7 +51,7 @@ function render(content: ContentUnion, props: RenderPropsUnion) {
 }
 
 function googleInbound(message: Message): ContentUnion {
-  const messageJson = message.content.message;
+  const messageJson = message.content.message ?? message.content;
```

Comment:
I'm not sure I understand how we got here? 🤔 If it's inbound the content should always have the same structure, right?

Reply:
Yes! Except when the inbound messages are a`SuggestionResponse` (the user clicks on a Suggestion and sends it as a message). In this case, messages have a different structure (`message.content.suggestionResponse.text` instead of `message.content.message.text`)